Makefile

```diff
@@ -11,9 +11,15 @@ RUN = poetry run
 SCHEMA_NAME = linkml_model
 SOURCE_SCHEMA_PATH = $(shell sh ./utils/get-value.sh source_schema_path)
 SRC = .
-DEST = target
-PYMODEL = linkml_model/
-DOCDIR = $(DEST)/docs
+DEST = staging
+PYMODEL = linkml_model
+DOCDIR = docs/source
```

Comment:
DOCDIR changes b/c of `mike` requirements?

Reply:
Sort of. Conceptually the "documentation" for `linkml-model` (i.e. what's accessible from `https://linkml.io/linkml-model`) needs to comprise the actual generated doc pages (served under the `/docs` route), the YAML source files (like `types.yaml`) and other generated artifacts (like `types.shex`). Currently we accomplish that by an unholy combination of running mkdocs _locally_ and GitHub Pages' automatic Jekyll configuration (tied to the `main` branch). But, yeah, `mike` needs to be part of a strictly mkdocs-based workflow. So that means we need to wrangle all the necessary files into place and then let `mkdocs`/`mike` process them. That's what `DOCDIR` is now.